### PR TITLE
metainfo_series: don't use title if series_name is set

### PR DESCRIPTION
**Pull request by cvium**
... probably not a  very good solution

Fixes https://github.com/Flexget/Flexget/issues/2071

### CODE REVIEW

File: flexget/plugins/metainfo/series.py [hunk 1]
```diff
@@ -43,7 +43,15 @@ def guess_entry(self, entry, allow_seasonless=False, config=None):
         identified_by = 'auto'
         if config and 'identified_by' in config:
             identified_by = config['identified_by']
-        parsed = get_plugin_by_name('parsing').instance.parse_series(data=entry['title'], identified_by=identified_by,
+        if entry.get('series_name'):
+            title = entry['series_name']
+            if entry.get('series_year'):
+                title += ' ({})'.format(entry['series_year'])
+            if entry.get('series_id'):
+                title += ' {}'.format(entry['series_id'])
```
Comment (liiight) [inline flexget/plugins/metainfo/series.py]: what's the use case for this? i don't remember what's series_id...

Comment (cvium) [reply flexget/plugins/metainfo/series.py]: I think it's the value of the ep identifier

**Comment by cvium** [comment]
@liiight @stevezau Where are we on this? Should we just push this?

**Comment by liiight** [comment]
I guess

**Pull request closed without merging**